include/rocksdb/db.h

```diff
@@ -1744,8 +1745,8 @@ class DB {
 
   // Trace block cache accesses. Use EndBlockCacheTrace() to stop tracing.
   virtual Status StartBlockCacheTrace(
-      const TraceOptions& /*options*/,
-      std::unique_ptr<TraceWriter>&& /*trace_writer*/) {
+      const BlockCacheTraceOptions& /*options*/,
```

Comment:
Can we also keep the original `StartBlockCacheTrace` and have it redirect to this implementation?